### PR TITLE
[NL Next] Fix Ranking Classifier Heuristics

### PR DESCRIPTION
**Pull request by juliawu**
This PR:
* Fixes the smaller vs larger mistake in current ranking classifiers
* Adds a few more heuristics to ranking classifiers

Tested by:
* Running new classifier on ~50 manually curated & GPT-3 generated queries
* Running new classifier on previous sample queries to check for false positives
* Ran through both test scripts, confirmed no change in correctness of outputs.

### CODE REVIEW

File: server/lib/nl/constants.py [hunk 1]
```diff
@@ -166,11 +166,14 @@
                 "best",  # leaving here for backwards-compatibility
                 "highest",
                 "high",
-                "smallest",
+                "largest",
```
Comment (shifucun) [inline server/lib/nl/constants.py:169]: Are there any tests for the classifier? Worth start to adding them?

Comment (juliawu) [reply server/lib/nl/constants.py:169]: Tests are a good idea. Should I add to this PR or should I submit tests in a followup PR?

Comment (shifucun) [reply server/lib/nl/constants.py:169]: Can add in this PR? There are some related tests can follow in https://github.com/datacommonsorg/website/tree/master/server/tests/lib/nl

Comment (juliawu) [reply server/lib/nl/constants.py:169]: Great, I'll convert to draft for now, and re-request review once tests are added.

Comment (shifucun) [reply server/lib/nl/constants.py:169]: Should be easy to add tests for heuristic classifications as it does not need mock or other setup?

Comment (juliawu) [reply server/lib/nl/constants.py:169]: Added tests!

Comment (shifucun) [reply server/lib/nl/constants.py:169]: very nice!

**Review by shifucun — APPROVED** [review]

**Pull request merged**